web/sites/default/settings.php

```diff
@@ -1,51 +0,0 @@
-<?php
```

Comment:
With this deleted, is it generated automatically by d9?

Reply:
Following suite from the last few projects, specifically Acquia hosted, that we've worked on.

Reply:
The docroot/sites/default/ equivalent is present. It still isn't there in the core download